apps/crn-server/src/controllers/working-group.controller.ts

```diff
@@ -5,7 +5,7 @@ import {
   WorkingGroupResponse,
 } from '@asap-hub/model';
 import { WorkingGroupDataProvider } from '../data-providers/types';
-import { toWorkingGroupResponse } from '../entities/working-group';
+import { toWorkingGroupResponse } from '../data-providers/entities';
```

Comment:
so the whole point of putting these transformers/parses inside the data-providers directory is that they are only used by data-providers so they should only parse from squidex/contentful to data objects and back. This is a controller helper/type. If the type is declared there move the type out

Reply:
the issue with this one, is the fact that it's also used in another "transformer" in the events one